docs/Cli.md

```diff
@@ -170,6 +171,66 @@ serial 0 -4
 
 `serial` can also be used without any argument to print the current configuration of all the serial ports.
 
+### `timer_output_mode`
+
+Since INAV 7, the firmware can dynamically allocate servo and motor outputs. This removes the need for bespoke targets for special cases (e.g. `MATEKF405` and `MATEKF405_S6SERVO`.
+
+#### Syntax
+
+```
+timer_output_mode [timer [function]]
+```
+where:
+* Without parameters, lists the current timers and modes
+* With just a `timer` lists the mode for that timer
+* With both `timer` and `function`, sets the function for that timers
+
+Note:
+
+* `timer` identifies the timer **index** (from 0); thus is one less than the corresponding `TIMn` definition in a target's `target.c`.
```

Comment:
Maybe we should mention the timer index shown in the mixer tab of the configurator. Probably easier for the user and less error prone than opening the target.c file to find the timer mapping.

Also, at32 chips call the timers TMRn.